api/python/provisioner/errors.py

```diff
@@ -132,6 +132,16 @@ def __str__(self):
         return repr(self.reason)
 
 
+class ValidationError(ProvisionerError, ValueError):
```

Comment:
I think my previous suggestion wasn't good enough: we don't need ValueError here since generally an error might not be related to wrong values. Please keep only ProvisionerError as a base class. Thanks

Reply:
Comment addressed.